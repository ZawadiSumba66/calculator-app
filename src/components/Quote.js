export default function Quote() {
  return (
    <div className="center">
      <h3>
        Mathematics is not about numbers, equations, computations, or algorithms: it
        is about understanding.
        <br />
        {' '}
        â€“William Paul Thurston
      </h3>
    </div>
  );
}
