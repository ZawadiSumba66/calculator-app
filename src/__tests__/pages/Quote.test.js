import renderer from 'react-test-renderer'
import Quote from '../../components/Quote'
import React from 'react'


it('matches the Quote snapshot',()=>{
   const tree= renderer.create(<Quote/>).toJSON()
   expect(tree).toMatchSnapshot()
})