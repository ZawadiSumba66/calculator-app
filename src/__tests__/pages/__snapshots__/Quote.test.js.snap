// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the Quote snapshot 1`] = `
<div
  className="center"
>
  <h3>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding.
    <br />
     
    â€“William Paul Thurston
  </h3>
</div>
`;
